### DISCUSSION TITLE
Close discussions?

### DISCUSSION THREAD

**Discussion by markus2330** [ElektraInitiative/libelektra#3821] [Archive Answers] [answered]
I think we already have a few things in the discussions which are not relevant anymore, e.g. https://github.com/ElektraInitiative/libelektra/discussions/3765

How can we close this? Do we need to delete such discussions?

**Comment by mpranj** [comment, marked as answer]
Last time I checked closing is not implemented 🙃. A suggested workaround is to put everything in an archive category.
Deleting sounds drastic, then some answers can't be found.

**Reply by markus2330** [reply]
I put this question now in "Archive Answers". How can we hide it now?

**Reply by mpranj** [reply]
We can not, afaik. Others are complaining about this missing functionality as well (see [feature request](https://github.com/github/feedback/discussions/3097)).

We can either put it in the "Archive" category as you have done, or completely delete the discussion (then nobody can look it up). It can not be hidden or marked as resolved. Only Q&A can be marked as answered, but it still stays open to further discussion.

**Reply by kodebach** [reply]
> Only Q&A can be marked as answered

Proposals as well. I think we can configure this per category.

> but it still stays open to further discussion.

There is a button in the sidebar to "Lock conversation"
